packages/data/src/components/use-select/index.ts

```diff
@@ -106,7 +109,7 @@ export default function useSelect( mapSelect, deps ) {
 	// `_mapSelect` if we can.
 	const callbackMapper = useCallback(
 		hasMappingFunction ? mapSelect : noop,
-		deps
+		deps ?? []
```

Comment:
This change is (weirdly) causing unit tests to fail 😅

Reply:
replaced with a static `noop` to fix the unit tests.

Reply:
Fixed